src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/ReadWrite/ByteTests.cs

```diff
@@ -119,20 +119,6 @@ public void ReadByte_StructWithReferenceTypes_ReturnsExpected()
             };
 
             Assert.Equal(100, Marshal.ReadByte(structure, pointerOffset));
```

Comment:
Actually, can you put in a comment here for why we aren't testing this scenario and a short description of what we tried.

Reply:
Added!